src/codeu/chat/server/Server.java

```diff
@@ -89,9 +89,13 @@ public void onMessage(InputStream in, OutputStream out) throws IOException {
             author,
             conversation,
             message.id));
+     if (type == NetworkCode.SERVER_INFO_REQUEST) {
```

Comment:
This isn't quite right, but it looks like the tutorial is a bit out of date. You should add a new command in this class  - your code should look something like 
```
     this.commands.put(NetworkCode. SERVER_INFO_REQUEST,  new Command() {
        @Override
       public void onMessage(InputStream in, OutputStream out) throws IOException {
        Serializers.INTEGER.write(out, NetworkCode.SERVER_INFO_RESPONSE);
        Uuid.SERIALIZER.write(out, info.version); 
        }
    }
```